admin/src/main/scala/io/buoyant/admin/AdminConfig.scala

```diff
@@ -1,15 +1,18 @@
 package io.buoyant.admin
 
 import io.buoyant.config.types.Port
-import java.net.InetSocketAddress
+import java.net.{InetAddress, InetSocketAddress}
 
 case class AdminConfig(
+  ip: Option[InetAddress],
   port: Option[Port],
```

Comment:
I think if we went the HostAndPort route then we would get rid of the `AdminConfig` class and then in `LinkerConfig` the `admin` field would simply be of type `HostAndPort`.

I could go either way on doing that or leaving it as is.